README.md

```diff
@@ -57,13 +58,29 @@ CORS can be configured in a `json` file that is pointed at from `package.json:ma
 
 ### bm-server.json
 
+By setting cors to `false`, Cross-Origin resource sharing will not be allowed. **This is the default behaviour**
+
+```json
+{
+  "cors": false
+}
+```
+
+By setting cors to `true`, defaults below will be used.
+
+```json
+{
+  "cors": true
```

Comment:
I saw `object | true | false` options as the best option considering we are trying to cater for the following:

- Majority of our customers will not be making cross origin requests because of native apps etc.
- We want to enforce best practices on our customers.

Very had to cater for both...